.github/workflows/acceptance-tests.yml

```diff
@@ -0,0 +1,123 @@
+name: Acceptance Tests
+
+on:
+  # So that we do a final check before publishing
+  pull_request_review:
+    types: [submitted]
+  # So that we don't do expensive tests until approved
+  push:
+    branches: [main]
```

Comment:
But, because we don't have a merge queue, we still want to test again on `main` before publishing to `@latest`